network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/share-buttons.html

```diff
@@ -0,0 +1,41 @@
+{% load i18n %}
+
+{% with request.scheme|add:'://'|add:request.get_host|add:request.get_full_path as current_url %}
+        <div class="share-button-group-wrapper" 
+             data-link="{{current_url}}">
+            <div class="share-button-group {% if version == 'mini' %} circle {% else %} rectangle {% endif %} {% if layout == 'stacked' %} stacked {% endif %}">
+                <div class="subgroup">
+                    <a class="btn btn-secondary btn-share facebook-share" href="https://www.facebook.com/sharer/sharer.php?u={{current_url|urlencode}}">
+                        {% if version == 'mini' %} 
+                          <span class="sr-only">Share on Facebook</span>
```

Comment:
Hi @TheoChevalier! I was wondering if we should wrap these in a trans block? 
```suggestion
                          <span class="sr-only">{% trans "Share on Facebook" %}</span>
```